rdbms/src/test/java/io/spine/server/storage/jdbc/query/AbstractQueryShould.java

```diff
@@ -65,10 +69,20 @@ public void close_connection() throws SQLException {
     }
 
     @Test
-    public void hold_cursors_over_commit() throws SQLException {
-        final Connection connection = query.factory()
-                                           .getConnection();
-        assertEquals(HOLD_CURSORS_OVER_COMMIT, connection.getHoldability());
+    public void set_hold_cursors_over_commit_for_connection() throws SQLException {
```

Comment:
This test requires some Javadoc, explaining what the test scenario is.

Reply:
Done.